cypress/integration/mocked/okta_sign_in.spec.ts

```diff
@@ -0,0 +1,175 @@
+describe('Sign in flow', () => {
+  context('Signing in - Okta', () => {
+    const defaultReturnUrl = 'https://m.code.dev-theguardian.com';
+
+    beforeEach(() => {
+      cy.mockPurge();
+    });
+
+    it('loads the redirectUrl if user is already authenticated', function () {
+      cy.mockPattern(
+        200,
+        {
+          id: 'test',
+          login: 'user@example.com',
+          userId: 'userId',
+          status: 'ACTIVE',
+          expiresAt: '2016-01-03T09:13:17.000Z',
+          lastPasswordVerification: '2016-01-03T07:02:00.000Z',
+          lastFactorVerification: null,
+          amr: ['pwd'],
+          idp: {
+            id: '01a2bcdef3GHIJKLMNOP',
+            type: 'OKTA',
+          },
+          mfaActive: true,
+        },
+        '/api/v1/sessions/the_sid_cookie',
+      );
+
+      cy.mockPattern(204, {}, '/api/v1/users/userId/sessions');
+
+      cy.setCookie('sid', `the_sid_cookie`);
+
+      // we can't actually check the authorization code flow
+      // so intercept the request and redirect to the guardian about page
+
+      cy.visit(
+        '/signin?returnUrl=https%3A%2F%2Fwww.theguardian.com%2Fabout&useOkta=true',
+      );
+
+      Cypress.on('uncaught:exception', (err, runnable) => {
+        return false;
```

Comment:
Do we mind warnings? If we do, maybe we should run eslint on the CI with `--max-warnings=0`.

Reply:
yeah this was actually to catch exceptions when visiting `theguardian.com`. I'd be happy to discuss being stricter on warnings, but maybe outside this PR ?

Reply:
I think the only action needed on this PR is to delete the unused vars `err` and `runnable`. There are no warnings from the linter output currently.

Reply:
ah sorry, I see : )